.fossa.yml

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Why do we need this file? Where is it being referenced from?

Reply:
Running fossa analyze references this as the main config file. If there is none, it will try to analyze everything in the repo. We only want to analyze the fossa-deps.yml file, so we need a config file to tell it to ignore all the other files.